src/universal/components/Navigation/Navigation.css

```diff
@@ -22,8 +22,15 @@
   font-weight: 500 !important;
   color: primary !important;
 }
+
 .brand {
   color: primary;
   text-decoration: none;
   font-size: 1.75em; /* ~28px */
 }
+
+.span {
```

Comment:
true, but I was trying to hack-a-way to get the text aligned properly with the icon... tiss WIP still ;)